pkg/runtime/runtime.go

```diff
@@ -440,6 +440,13 @@ func (a *DaprRuntime) setupTracing(ctx context.Context, hostAddress string, tpSt
 	return nil
 }
 
+func getOtelServiceName(fallback string) string {
+	if value, ok := os.LookupEnv("OTEL_SERVICE_NAME"); ok {
```

Comment:
Ideally we would have a `DAPR_` prefix to namespace this environment variable to the project, but looks like there are other non-namespaced OTEL vars in use today so that ship has sailed.

Reply:
This is a standard env var: https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/#general-sdk-configuration